src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -454,6 +469,7 @@ export const CheckoutPayment = ({
   }, [checkoutStatus, completeCheckout])
 
   if (checkoutStatus === CheckoutStatus.Completed) {
+    localStorage.removeItem('paymentConnected')
```

Comment:
The render function should be pure.. but with this we are introducing side effects.

Can we do this in an event handler?